README.md

```diff
@@ -6,156 +6,39 @@
 [![Downloads](https://img.shields.io/docker/pulls/otel/demo)](https://hub.docker.com/r/otel/demo)
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg?color=red)](https://github.com/open-telemetry/opentelemetry-demo/blob/main/LICENSE)
 
-## Under Construction
+## Welcome to the OpenTelemetry Community Demo
```

Comment:
I'd add "(Astronomy Shop)" to the end of the heading because this name is used way below without any introduction.

Reply:
Added it as as part of the title directly